pkg/api/pod/util.go

```diff
@@ -386,6 +386,8 @@ func GetValidationOptionsFromPodSpecAndMeta(podSpec, oldPodSpec *api.PodSpec, po
 		AllowNamespacedSysctlsForHostNetAndHostIPC:        false,
 		AllowNonLocalProjectedTokenPath:                   false,
 		AllowPodLifecycleSleepActionZeroValue:             utilfeature.DefaultFeatureGate.Enabled(features.PodLifecycleSleepActionAllowZero),
+		InPlacePodVerticalScalingEnabled:                  utilfeature.DefaultFeatureGate.Enabled(features.InPlacePodVerticalScaling),
```

Comment:
This doesn't seem related to your feature.

Reply:
fixed